specs-jetpack-calypso/wp-jetpack-sso-connect-spec.js

```diff
@@ -0,0 +1,129 @@
+import test from 'selenium-webdriver/testing';
+import { By } from 'selenium-webdriver';
+import config from 'config';
+
+import JetpackAuthorizePage from '../lib/pages/jetpack-authorize-page';
+import WPAdminSidebar from '../lib/pages/wp-admin/wp-admin-sidebar.js';
+import LoginFlow from '../lib/flows/login-flow';
+import SidebarComponent from '../lib/components/sidebar-component';
+import JetpackConnectFlow from '../lib/flows/jetpack-connect-flow';
+
+import * as driverManager from '../lib/driver-manager';
+import * as driverHelper from '../lib/driver-helper';
+import * as dataHelper from '../lib/data-helper';
+import SignUpFlow from '../lib/flows/sign-up-flow';
+import WPAdminDashboardPage from '../lib/pages/wp-admin/wp-admin-dashboard-page';
+import WPAdminNewUserPage from '../lib/pages/wp-admin/wp-admin-new-user-page';
+import WPAdminLogonPage from '../lib/pages/wp-admin/wp-admin-logon-page';
+import EmailClient from '../lib/email-client';
+
+const mochaTimeOut = config.get( 'mochaTimeoutMS' );
+const startBrowserTimeoutMS = config.get( 'startBrowserTimeoutMS' );
+const signupInboxId = config.get( 'signupInboxId' );
+const screenSize = driverManager.currentScreenSize();
+const host = dataHelper.getJetpackHost();
+const userAccount = 'jetpackUser' + host;
+
+let driver;
+let blogCreds;
+
+test.before( function() {
+	this.timeout( startBrowserTimeoutMS );
+	driver = driverManager.startBrowser();
+} );
+
+test.describe( `Jetpack Connect: (${ screenSize })`, function() {
+	this.timeout( mochaTimeOut * 2 );
+
+	test.describe( 'Disconnect expired sites: @parallel @jetpack', function() {
+		this.bailSuite( true );
+
+		test.before( function() {
+			return driverManager.clearCookiesAndDeleteLocalStorage( driver );
+		} );
+
+		test.it( 'Can log in', () => {
+			const loginFlow = new LoginFlow( driver, userAccount );
+			loginFlow.loginAndSelectMySite();
+		} );
+
+		test.it( 'Can disconnect any expired sites', () => {
+			this.sidebarComponent = new SidebarComponent( driver );
+
+			const removeSites = () => {
```

Comment:
Would be good to factor this `removeSites` function out so that it can be shared with the [jetpack connect spec](https://github.com/Automattic/wp-e2e-tests/blob/68848cd1a0b8fc3f4056fdb9484b3abf6cded316/specs-jetpack-calypso/wp-jetpack-connect-spec.js#L52). That can be for a future PR because this one is large already.